core/src/Cuda/Kokkos_Cuda_KernelLaunch.hpp

```diff
@@ -303,6 +305,45 @@ struct CudaParallelLaunchKernelInvoker<
          get_kernel_func())<<<grid, block, shmem, cuda_instance->m_stream>>>(
         driver);
   }
+
+#ifdef KOKKOS_CUDA_ENABLE_GRAPHS
+  inline static void create_parallel_launch_graph_node(
```

Comment:
No, because it will need to do more than just that in certain cases, and I don't think I should create separate customization points for the other things that may need to happen.